### ISSUE TITLE
[flaky CI] [rfe_id:588]ContainerDisk [rfe_id:273]Starting multiple VMIs with ephemeral registry disk [test_id:1465]should success

### ISSUE THREAD

**Issue by AlonaKaplan** [kubevirt/kubevirt#3661]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
kubevirt-e2e-k8s-1.16 lane -
[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:component]ContainerDisk [rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting multiple VMIs with ephemeral registry disk [test_id:1465]should success

/root/go/src/kubevirt.io/kubevirt/tests/container_disk_test.go:147
Timed out after 180.000s.
Timed out waiting for VMI testvmiz45vw66zzsd2x48bnv4fmx89lqr2slf82pdt8xjtjt52bzqg to enter [Running] phase(s)
Expected
    <v1.VirtualMachineInstancePhase>: Scheduled
to be an element of
    <[]interface {} | len:1, cap:1>: [["Running"]]
/root/go/src/kubevirt.io/kubevirt/tests/utils.go:2614

Link to the failing job: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3502/pull-kubevirt-e2e-k8s-1.16/1275550270280962048

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by rmohr** [comment]
/cc @davidvossel 

I think this one is a real bug.

The logs show the pods running https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3502/pull-kubevirt-e2e-k8s-1.16/1275550270280962048/artifacts/2_pods.log:

```
            "status": {
                "phase": "Running",
                "conditions": [
                    {
                        "type": "Initialized",
                        "status": "True",
                        "lastProbeTime": null,
                        "lastTransitionTime": "2020-06-24T00:51:39Z"
                    },
                    {
                        "type": "Ready",
                        "status": "True",
                        "lastProbeTime": null,
                        "lastTransitionTime": "2020-06-24T00:51:49Z"
                    },
                    {
                        "type": "ContainersReady",
                        "status": "True",
                        "lastProbeTime": null,
                        "lastTransitionTime": "2020-06-24T00:51:49Z"
                    },
                    {
                        "type": "PodScheduled",
                        "status": "True",
                        "lastProbeTime": null,
                        "lastTransitionTime": "2020-06-24T00:51:39Z"
                    }
                ],
```

and virt-handler loops in https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3502/pull-kubevirt-e2e-k8s-1.16/1275550270280962048/artifacts/pods/2_kubevirt_virt-handler-8jtb2-virt-handler.log on:


```
{"component":"virt-handler","kind":"","level":"info","msg":"VMI is in phase: Scheduled\n","name":"testvmiz45vw66zzsd2x48bnv4fmx89lqr2slf82pdt8xjtjt52bzqg","namespace":"kubevirt-test-default","pos":"vm.go:1123","timestamp":"2020-06-24T00:55:57.822144Z","uid":"5fd6e797-85d1-427d-8d84-d12d58f58b9c"}
{"component":"virt-handler","kind":"Domain","level":"info","msg":"Domain status: Running, reason: Unknown\n","name":"testvmiz45vw66zzsd2x48bnv4fmx89lqr2slf82pdt8xjtjt52bzqg","namespace":"kubevirt-test-default","pos":"vm.go:1128","timestamp":"2020-06-24T00:55:57.822287Z","uid":"5fd6e797-85d1-427d-8d84-d12d58f58b9c"}
{"component":"virt-handler","level":"error","msg":"failed to read from cache file: open /var/run/kubevirt-private/network-info-cache/5fd6e797-85d1-427d-8d84-d12d58f58b9c/default: no such file or directory","pos":"vm.go:415","reason":"open /var/run/kubevirt-private/network-info-cache/5fd6e797-85d1-427d-8d84-d12d58f58b9c/default: no such file or directory","timestamp":"2020-06-24T00:55:57.822662Z"}
```

To summarize: Pod is running, the domain is running but the VMI is not updated to `Running` from `Scheduled` due to 

```
{"component":"virt-handler","level":"error","msg":"failed to read from cache file: open /var/run/kubevirt-private/network-info-cache/5fd6e797-85d1-427d-8d84-d12d58f58b9c/default: no such file or directory","pos":"vm.go:415","reason":"open /var/run/kubevirt-private/network-info-cache/5fd6e797-85d1-427d-8d84-d12d58f58b9c/default: no such file or directory","timestamp":"2020-06-24T00:55:57.822662Z"}
```

**Comment by AlonaKaplan** [comment]
The [PR](https://github.com/kubevirt/kubevirt/pull/3488) adding this dir  /var/run/kubevirt-private/network-info-cache was [reverted](https://github.com/kubevirt/kubevirt/pull/3641).

/cc @vatsalparekh please make sure the new [PR](https://github.com/kubevirt/kubevirt/pull/3642) doesn't introduce a bug.

**Comment by rmohr** [comment]
/close
@AlonaKaplan thanks.

Closing since it is not in the codebase and the test looks very stable: https://storage.googleapis.com/kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flakefinder-2020-06-25-168h.html#row145
